src/main/java/com/minecolonies/colony/buildings/BuildingFarmer.java

```diff
@@ -325,7 +344,9 @@ public void syncWithColony(@NotNull World world)
                 else
                 {
                     scarecrow.setName(LanguageHandler.format("com.minecolonies.gui.scarecrow.user", getWorker().getName()));
-                    getColony().getWorld().markBlockForUpdate(scarecrow.getPos());
+                    getColony().getWorld()
+                      .notifyBlockUpdate(scarecrow.getPos(), getColony().getWorld().getBlockState(scarecrow.getPos()), getColony().getWorld().getBlockState(scarecrow
+                                                                                                                                                              .getPos()), 3);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 3 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)